tools/release-jobs-syncer/pkg/release.go

```diff
@@ -113,16 +113,18 @@ func updateProwJobsForReleases(gc ghutil.GithubOperations, orgRepoReleaseMap map
 		sortReleases(releases)
 		log.Printf("Existing releases for %s/%s: %v", org, repo, releases)
 		log.Printf("Latest release for %s/%s: %s", org, repo, latest)
-		// Skip if the latest release has already been configured.
+		// Set latest to nothing if the latest release has already been configured.
```

Comment:
This comment is hard to understand. Maybe replace the whole line with your justification from PR.

Reply:
reorged it a little. The continue in the old code was the problem.